src/packages.builds

```diff
@@ -15,5 +16,17 @@
     </Project>
   </ItemGroup>
 
+  <UsingTask TaskName="GenerateNetStandardSupportTable" AssemblyFile="$(PackagingTaskDir)Microsoft.DotNet.Build.Tasks.Packaging.dll" />
```

Comment:
I assume this target is manual so we would need to directly call this manually have a full package build?